src/slider/index.html

```diff
@@ -0,0 +1,5 @@
+<div ref="base" class="gb-slider__base" mousedown="{ onClick }">
+  <div ref="lower" class="gb-slider__handle gb-slider__handle-lower" style="left: 0%"  mousedown="{ onDragStart }" touchstart="{ onDragStart }"></div>
```

Comment:
can we make the class `gb-slider__handle--lower` and `...--upper` for a modifier?